proposals/csharp-8.0/patterns.md

```diff
@@ -50,19 +50,23 @@ var_pattern
     : 'var' designation
     ;
 positional_pattern
-    : type? '(' subpatterns? ')' property_subpattern? simple_designation?
-    ;
-subpatterns
+    : type? '(' positional_subpatterns? ')' property_subpattern? simple_designation?
+    ; 
+positional_subpatterns
     : subpattern
```

Comment:
It seems to me like this could just be expressed by changing this like to:
```suggestion
    : subpattern ','?
```
Is there something I'm missing that requires the rest of these changes?

Reply:
Oh, I didn't think I committed the other changes! Will revert them.

Reply:
reverted

Reply:
Actually changed it slightly so that it's consistent with section lower down in the spec, and distinguishes between positional and property patterns.